nc/admin.py

```diff
@@ -70,6 +80,7 @@ class ResourceAdmin(admin.ModelAdmin):
     filter_horizontal = ("agencies",)
     inlines = [InlineResourceFile]
     readonly_fields = ("created_date",)
+    form = ResourceForm
```

Comment:
[`formfield_overrides`](https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_overrides) might work here, but not required!